bin/next

```diff
@@ -49,19 +54,21 @@ if (new Set(['--help', '-h']).has(cmd)) {
 }
 
 if (commands.has(cmd)) {
-  args = process.argv.slice(3)
+  args = args.concat(process.argv.slice(3))
 } else {
   cmd = defaultCommand
-  args = process.argv.slice(2)
+  args = args.concat(process.argv.slice(2))
 }
 
 const defaultEnv = cmd === 'dev' ? 'development' : 'production'
 process.env.NODE_ENV = process.env.NODE_ENV || defaultEnv
 
+const node = process.argv[0]
 const bin = join(__dirname, 'next-' + cmd)
+args.push(bin)
 
 const startProcess = () => {
-  const proc = spawn(bin, args, { stdio: 'inherit', customFds: [0, 1, 2] })
+  const proc = spawn(node, args, { stdio: 'inherit', customFds: [0, 1, 2] })
```

Comment:
Shouldn't this be `bin`? = `const bin = join(__dirname, 'next-' + cmd)`

Reply:
No. I've changed the scripts to be regular JavaScript files, that doesn't contain `#!/usr/bin/env node` as we need to launch Node with the `--inspect` flag and having the files as a shell script doesn't allow that.

So the change here is the that we spawn a child process like `node <path to next-dev>`